core/templates/dev/head/pages/exploration_player/conversation_skin_directive.html

```diff
@@ -246,7 +246,7 @@
       the same line-height, margin-top and margin-bottom attributes)
   */
   .rte-viewer > p {
-    line-height: 28px;
+    line-height: 1.846;
```

Comment:
Do we have to keep this in sync with `.form-control.oppia-rte-content > div > p`? (as mentioned above in line 245)

(Ignore if `.oppia-rte-content > div > p` === `.form-control.oppia-rte-content > div > p`)

Reply:
@bansalnitish, Found `.form-control.oppia-rte-content > div > p` on line 1515 and 1518 in oppia.css

Reply:
@DubeySandeep, I don't think we have to alter anything in that section.